### ISSUE TITLE
Find-PSResource -CommandName is case-sensitive

### ISSUE THREAD

**Issue by sdwheeler** [PowerShell/PSResourceGet#920]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://www.powershellgallery.com/packages/PowerShellGet)
- [X] Search the existing issues.

### Steps to reproduce

When searching for a resource by **CommandName** you must specify the name using the correct case.

```
Find-PSResource -CommandName new-markdownhelp -ModuleName platyps
Find-PSResource -CommandName New-MarkdownHelp -ModuleName platyps

Name             ParentResource
----             --------------
New-MarkdownHelp Microsoft.PowerShell.PowerShellGet.UtilClasses.PSResourceInfo
```

### Expected behavior

```console
The search should be case-insensitive.
```

### Actual behavior

```console
You must know the case of the command name.

This can be a problem when you don't know how the module author named the command. This could be especially problematic for Az cmdlets that don't always follow the Pascal-case naming pattern.
```

### Environment data

```powershell
$PSVersionTable.PSVersion

Major  Minor  Patch  PreReleaseLabel BuildLabel
-----  -----  -----  --------------- ----------
7      3      2

ModuleType Version    PreRelease Name                                ExportedCommands
---------- -------    ---------- ----                                ----------------
Binary     3.0.19     beta19     powershellget                       {Find-PSResource, Get-PSResource, Get-PSResourceR…
```

**Comment by SydneyhSmith** [comment]
Fixed by #1023

**Comment by sdwheeler** [comment]
Confirmed
